runtime/flang/CMakeLists.txt

```diff
@@ -415,24 +417,39 @@ SET(FTN_SUPPORT_DESC_DEP
   )
 
 set(I8_FILES_DIR I8_sources)
+separate_arguments(SEPARATED_CMAKE_Fortran_FLAGS NATIVE_COMMAND ${CMAKE_Fortran_FLAGS})
```

Comment:
@gklimowicz This requires at least cmake 3.9. Can the minimum required in flang/CMakeLists.txt be appropriately changed?
https://cmake.org/cmake/help/v3.9/release/3.9.html#commands